tests/runtest

```diff
@@ -0,0 +1,36 @@
+SHELL := /bin/bash
+PYTHON_VERSIONS := 3.9 3.10 3.11 3.12 3.13.2
+REQUIREMENTS := test_requirements.txt
+PYENV_EXISTS := $(shell command -v pyenv)
+
+.PHONY: all install_pyenv install_requirements install_python_versions set_global_versions run_nox
+
+all: install_pyenv install_requirements install_python_versions set_global_versions run_nox
+
+install_pyenv:
+ifeq ($(PYENV_EXISTS),)
+	@echo "Installing pyenv..."
+	@curl https://pyenv.run | bash
+else
+	@echo "Pyenv is already installed."
+endif
+
+install_requirements:
+	@echo "Installing test_requirements..."
+	@pip install -r $(REQUIREMENTS)
+	@echo "Requirements installed."
+
+install_python_versions:
+	@echo "Installing Python versions..."
+	@for version in $(PYTHON_VERSIONS); do pyenv install $$version || true; done
+	@echo "Python versions installed."
+
+set_global_versions:
+	@echo "Setting global Python versions..."
+	@pyenv global $(PYTHON_VERSIONS)
+	@echo "Global Python versions set."
+
+run_nox:
```

Comment:
This can be simply named `tests` so running `make tests` runs tests. Nox is just a tool, an implementation detail. Potential contributors don't necessarily need to know about it.